app/uk/gov/hmrc/transitmovements/controllers/errors/ConvertError.scala

```diff
@@ -67,6 +68,16 @@ trait ConvertError {
     }
   }
 
+  implicit val objectStoreErrorConverter = new Converter[ObjectStoreError] {
+
+    import uk.gov.hmrc.transitmovements.services.errors.ObjectStoreError._
+
+    def convert(objectStoreError: ObjectStoreError): PresentationError = objectStoreError match {
+      case FileNotFound(fileLocation) => PresentationError.notFoundError(s"file not found at location: $fileLocation")
```

Comment:
Would this be an internal server error instead?

Reply:
Bad request. The header provided by the calling service is bad, because it doesn't point to a file. Not this service's fault.